doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -110,13 +136,40 @@ ssh into the dataverse-internal server and download the release candidate war fi
 
 Go to /doc/release-notes, open the release-notes.md file for the release we're working on, and perform all the steps under "Upgrade Instructions". Note that for regular releases, we haven't bumped the version yet so you won't be able to follow the steps exactly. (For hotfix releases, the version will be bumped already.)
 
+Deploy Release Candidate to QA
+------------------------------
+
+|dedicated|
+
+Deploy the same war file to https://qa.dataverse.org using the same upgrade instructions as above.
+
 Deploy Release Candidate to Demo
```

Comment:
At the retrospective, I have half a memory that we might want to stop deploying a release candidate to demo. Is that right? If so, I'm happy to delete this step.

Do note that we have a "Deploy Final Release on Demo" later.